packages/react-search-ui-views/src/types/Result.js

```diff
@@ -2,6 +2,15 @@ import PropTypes from "prop-types";
 
 import FieldValueWrapper from "./FieldValueWrapper";
 
-// An object where keys are the field names, and values are the values
-// corresponding to that
-export default PropTypes.objectOf(FieldValueWrapper);
+// Typically an object where keys are the field names, and values are field values.
+// Also could be literally any other arbitrary value depending on the particular Search API response.
+// Default views in Search UI know what to do with FieldValueWrapper values, but not arbitrary values, so it
+// is typically better to work with FieldValueWrapper values. So we encourage FieldValueWrapper, but we accept
```

Comment:
v minor copy thing, I'd suggest replacing `typically` here with `usually` or `generally`, since we've already used `typically` at the start of the comment.

I'd also lose the `So` as well since we already used 'so' just one sentence ago, and the sentence reads fine without it.

```suggestion
// is usually better to work with FieldValueWrapper values. We encourage FieldValueWrapper, but we accept
```